app/assets/stylesheets/responsive.scss

```diff
@@ -1,2 +1 @@
-@import "bootstrap-responsive";
 @import "bootstrap-responsive-overrides";
```

Comment:
Hmm. I was thinking about that... yeah, and I'll split them out again if needed.